spec/reek/smell_detectors/irresponsible_module_spec.rb

```diff
@@ -49,17 +49,6 @@ class Charlie
       expect(src).not_to reek_of(:IrresponsibleModule)
     end
 
-    it "does not report re-opened #{scope} in the same file" do
-      src = <<-EOS
-        # This comment describes Alfa
-        #{scope} Alfa; end
-
-        #{scope} Alfa; def bravo; end; end
-      EOS
-
-      expect(src).not_to reek_of(:IrresponsibleModule)
-    end
```

Comment:
This change makes it impossible for detectors to keep state between sniffing one context and the next. Therefore, the behavior specified here can no longer work. I don't think this is a great loss, since:

* Why would you open a class or module twice in a single file, *and* define methods in it both times?
* None of the other detectors keep state for the same kind of case, e.g., TooManyMethods, and UnusedPrivateMethod.